src/formBuilder/radio/FBRadioButton.js

```diff
@@ -26,11 +26,12 @@ export default function FBRadioButton(props: Props) {
     disabled,
     autoFocus,
   } = props;
+  const [id] = useState(`radio-${Math.floor(Math.random() * 1000000)}`);
```

Comment:
Multiplication by 1000000 + flooring to clear out the decimal point, it being a forbidden character?

(My preferred approach is `Math.random().toString().slice(2)`, but to each his own!)

Reply:
yup!